### PR TITLE
traduction in french

### PR DESCRIPTION
**Pull request by MonsieurBibo**
I tried to translate this index.html in french. If you feel like something is wrong or weird, feel free to send me a message! Also you can use this file freely but it will be pretty nice from you if you can mentioned me on the french translation ;)

### CODE REVIEW

File: fr/index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      reliées à des langages ou des librairies que je n'ai jamais utilisées,
+      car la réponse relève (pour le développeur) du bon sens.
+    </p>
+    <p>
+      Vous pouvez aussi être vu comme quelqu'un...
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        J'ai une question à propos de Java mais je suis trop fatigué pour formuler
```
Comment (painor) [inline fr/index.html]: is `fatigué` the correct word here ?. The user isn't tired he is just lazy

File: fr/index.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+  <title>Ne demande pas la permission de demander, demande juste.</title>
```
Comment (painor) [inline fr/index.html]: I am not so sure about this translation as well. the `just` in `just ask` is a synonym to `simply ask` not to `ask correctly`

**Comment by tulir** [comment]
> it will be pretty nice from you if you can mentioned me on the french translation

You can add a link to your github profile/website/whatever in the footer similar to this: https://github.com/maunium/dontasktoask.com/blob/master/ar/index.html#L140

File: fr/index.html (hunk 1 above)
Comment (MonsieurBibo) [reply fr/index.html]: hey, thanks for the review. I think both work. "Fatigué" means tired but can also be used to say "lazy" sometimes.

File: fr/index.html (hunk 2 above)
Comment (MonsieurBibo) [reply fr/index.html]: The sentence cannot be perfectly translated, but I think this the best translation. I will ask my friends if they have another idea.

**Pull request merged**